CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 All notable changes to [Solang](https://github.com/hyperledger-labs/solang/)
 will be documented here.
 
+## [Unreleased]
+
+### Added
+- Added spl-token integration for Solana
+
+### Changed
+- The documentation has been re-arranged for readability.
+
+### Fixed
+- Fixed many parser issues. Now solang-parser parses all files in the
+  Ethereum Solidity test suite when `cargo test --workspace` is run.
```

Comment:
It is worth mentioning somewhere that we need to run `git submodule update --init --recursive` in order for this new test to work.